docs/external-disk.md

```diff
@@ -19,45 +19,112 @@ The `-disk` specification may be repeated for multiple disks, although a limited
 
 **TODO:** GCP
 
-## Mount the Disk
-A disk created or used via `hyperkit run` will be available inside the image at `/dev/vda` with the first partition at `/dev/vda1`.
+## Format the disk
 
-In order to use the disk, you need to do several steps to make it available:
+`pkg/format` creates a partition table and format drives for use with LinuxKit
 
-1. Create a partition table if it does not have one.
-2. Create a filesystem if it does not have one.
-3. `fsck` the filesystem.
-4. Mount it.
+### Example Usage
 
-To simplify the process, two `onboot` images are available for you to use:
+This packages supports two modes of use:
 
-1. `format`, which:
-    * checks for a partition table and creates one if necessary
-    * checks for a filesystem on the partition and creates one if necessary
-    * runs `fsck` on the filesystem
-2. `mount` which mounts the filesystem to a provided path
+```
+onboot:
+  name: format
+  image: linuxkit/format:<hash>
+```
+
+In this mode of operation, the first disk found that does not have a valid partition table
+will have one linux partition created that fills the entire disk
+
+### Options
+
+```
+onboot:
+  name: format
+  image: linuxkit/format:<hash>
+  command: ["/usr/bin/format", "-type", "ext4", "-label", "DATA", "/dev/vda"]
+```
+
+`-type` can be used to specify the type. This is `ext4` by default but `btrfs` and `xfs` are also supported
+`-label` can be used to give the disk a label
+The final (optional) argument specifies the device name
+
+## Mount the disk
+
+Once a disk has been prepared it will need to be mounted using `pkg/mount`
 
-```yml
+### Usage
+
+**NOTE: Block devices may only be mounted in `/var` unless you have explicitly added an additional bind mount**
+
+In this mode of operation, the first unmounted linux partition on the any block device is mounted to the mountpoint provided.
```

Comment:
"In this mode"..it doesn't say that there are multiple modes for mounting...

Maybe:
The simplest mode is ...